v2/protocol/http/protocol.go

```diff
@@ -70,6 +70,20 @@ type Protocol struct {
 	// If 0, DefaultShutdownTimeout is used.
 	ShutdownTimeout time.Duration
 
+	// ReadTimeout defines the http.Service ReadTimeout
+	// It is the maximum duration for reading the entire
+	// request, including the body. A zero or negative value means
+	// there will be no timeout.
+	ReadTimeout time.Duration
```

Comment:
question: do we need those public? I know ShutdownTimeout is, but IMHO doesn't need to be (would be breaking change)

Reply:
@embano1 I don't think so, I was just folloing what was done for ShutdownTimeout.

QQ: What does this break if its public? What would be the difference?